### PR TITLE
More from Timo's list.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: publications-2017.bib [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+@article{Maier2017,
+  doi = {10.1016/j.jcp.2017.03.014},
+  url = {https://doi.org/10.1016/j.jcp.2017.03.014},
+  year = {2017},
+  month = jun,
+  publisher = {Elsevier {BV}},
+  volume = {339},
+  pages = {126--145},
+  author = {Matthias Maier and Dionisios Margetis and Mitchell Luskin},
```
Comment (tjhei) [inline publications-2017.bib]: looks like @tamiko is not putting his papers here. :(

File: publications-2016.bib [hunk 2]
```diff
@@ -1,5 +1,71 @@
 % Encoding: US-ASCII
 
+@article{Kanschat2016,
+  doi = {10.1515/cmam-2016-0023},
```
Comment (tjhei) [inline publications-2016.bib]: already in publications-2017 as @article{2017_64, maybe delete that one?

File: publications-2017.bib (hunk 1 above)
Comment (tjhei) [reply publications-2017.bib]: edit: not true, this one is already in this file as @article{2017_81,

**Comment by bangerth** [comment]
Edits made. I appreciate the help finding duplicates -- though my plan had always been to just add everything I can find, and then when I have the list on the website sorted nicely by author, find the duplicates and discard. Either way, two less to worry about now :-)

**Comment by tjhei** [comment]
> Either way, two less to worry about now :-)

awesome, thanks for all your work on this! The graph is starting to look really good. :-)

**Pull request merged**